plugins/chucknorris/fortunes/chucknorris

```diff
@@ -228,7 +228,7 @@ Chuck Norris once punched the ground to stop an earthquake. The resulting afters
 %
 Chuck Norris once round-house kicked a salesman. Over the phone.
 %
-Chuck Norris once rounhouse kicked a football. The astronomical society now considers it a planet.
+Chuck Norris once roundhouse kicked a football. The astronomical society now considers it a planet.
```

Comment:
I suppose `round-house` might be better to match the preceding line.

Reply:
It doesn't matter, can be merged anyway.